core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -537,6 +539,14 @@ private boolean connectDetailsMatch(URI uri, org.apache.hadoop.conf.Configuratio
     // Alluxio Configuration.
     AlluxioConfiguration alluxioConf = new InstancedConfiguration(Configuration.global());
```

Comment:
is this a temporary conf? Is the host and port not needed elsewhere/later?

Reply:
This method just determines whether the connection details implied by the URI + HDFS configuration are different from the current connection details used by the FileSystemContext singleton. If they don't match, the global configuration will be properly updated elsewhere

Reply:
I see, so somewhere else, we already use `uri.getHost()` if it is there, and this method is just for checking? If that is the case, then shouldn't that other place update the conf correctly, to use the `uri.getHost()`?

Reply:
we check for changes in one place and then make the changes in another